pkgs/applications/misc/ddgr/default.nix

```diff
@@ -1,29 +1,17 @@
 {stdenv, fetchpatch, fetchFromGitHub, python3Packages}:
```

Comment:
```suggestion
{stdenv, fetchpatch, fetchFromGitHub, python3}:
```

Reply:
I thought about making this change, but I thought that keeping the argument passed in as `python3Packages` might help avoid breakage of anyone manually passing arguments into this derivation. It's also possible that I'm mistaken. Happy to make the change if that's desired.